src/features/broker-protection/actions/extract.js

```diff
@@ -152,13 +152,7 @@ function findFromElements (elements, key, extractField) {
     const elementValues = []
     if (elements) {
         for (const element of elements) {
-            let elementValue = rules[key]?.(element) ?? element?.innerText ?? null
```

Comment:
Weird, I hadn't even noticed that we were duplicating this logic in `findFromElement`. 🤦